.prettierrc.json

```
@@ -0,0 +1,6 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
```

I suggest removing the indentation inside the curly braces `{}` by analogy with the square brackets `[]`.

```suggestion
    "trailingComma": "all",
    "bracketSpacing": false,
```

---

Uhm, I'd prefer spaced `{}` :)